FEM/fem_ele_vec.cpp

```diff
@@ -2435,7 +2453,134 @@ void CFiniteElementVec::GlobalAssembly_RHS()
 				De->multi(strain_ne, dstress);
 				for (i = 0; i < ns; i++) // JT: This was commented. It shouldn't be.
 					dstrain[i] += strain_ne[i];
+		}
+			if (smat->Creep_mode == 1001) //BURGERS.
+			{
+			//get total strains at time t and current iteration i
+			double *strain_t, *strain_curr;
```

Comment:
why not use `std::vector<double> strain_t(6)`? with this, there is no need to write delete afterward